presto-geospatial-toolkit/src/test/java/com/facebook/presto/geospatial/serde/TestGeometrySerialization.java

```diff
@@ -133,16 +133,17 @@ public void testGeometryCollection()
     @Test
     public void testEnvelope()
     {
+        testEnvelopeSerialization(new Envelope());
```

Comment:
It looks like you are also fixing something here. Do you think it is worth extracing in a separate commit with an explanation?

Reply:
The hidden admixture is that I'm also allowing empty envelopes to be serialized.  I can separate that out into another commit.